src/google/protobuf/stubs/stringprintf.cc

```diff
@@ -150,10 +150,10 @@ std::string StringPrintfVector(const char* format,
   // or displaying random chunks of memory to users.
 
   const char* cstr[kStringPrintfVectorMaxArgs];
-  for (int i = 0; i < v.size(); ++i) {
+  for (unsigned int i = 0; i < v.size(); ++i) {
```

Comment:
This file has a couple more cases of `unsigned`.

Reply:
Woops, done